.github/workflows/lint.yaml

```diff
@@ -23,6 +23,10 @@ jobs:
           # list of changed files within `super-linter`
           fetch-depth: 0
 
+      - name: Disable renv
+        shell: bash
+        run: rm etl/.Rprofile
```

Comment:
Allows CI to pass.

Reply:
This was a Dan suggestion after I encountered [this](https://github.com/ccao-data/data-architecture/actions/runs/12360249252/job/34501973749).

Reply:
This is definitely a weird and ephemeral bug, but basically the R in superlinter will load `renv` due to the presence of the `.Rprofile` file in the working directory. The linter then fails because the `renv` environment doesn't have `lintr` in it. Removing the `.Rprofile` file loads the default superlinter R environment.

@wrridgeway You should add a comment to this step to explain why it's necessary/provide some context.